browser/ntp_background/ntp_p3a_helper_impl.cc

```diff
@@ -132,7 +131,7 @@ void NTPP3AHelperImpl::OnP3ARotation(p3a::MetricLogType log_type,
       BuildHistogramName(kCreativeTotalInstanceId, kCreativeTotalCountEventKey);
   // Always send the creative total if ads are disabled (as per spec),
   // or send the total if there were outstanding events sent
-  if ((ads_service_ != nullptr && !ads_service_->IsEnabled()) ||
+  if (!prefs_->GetBoolean(brave_ads::prefs::kOptedInToNotificationAds) ||
```

Comment:
we are sending NTP metrics here, so should it be `kNewTabPageShowSponsoredImagesBackgroundImage` preference?

Reply:
I believe this is correct because we want to know if push notification ads are enabled. cc @DJAndries